source/features/set-default-repositories-type-to-sources.tsx

```diff
@@ -10,13 +10,23 @@ function addSourceTypeToLink(link: HTMLAnchorElement): void {
 // No `include`, no `signal` necessary
 function init(): void {
 	observe([
-		'.header-nav-current-user ~ a[href$="tab=repositories"]', // "Your repositories" in the header profile dropdown
-		'[aria-label="User profile"] a[href$="tab=repositories"]', // "Repositories" tab on user profile
-		'[aria-label="Organization"] [data-tab-item="org-header-repositories-tab"] a', // "Repositories" tab on organization profile
+		`a:is([href^="/"], [href^="${location.origin}"])[href$="?tab=repositories"]`, // "Repositories" tab on user profile, only matches relative or same-origin links
+		`a:is([href^="/orgs/"], [href^="${location.origin}/orgs/"])[href$="/repositories"]`, // "Repositories" tab on organization profile, only matches relative or same-origin links
+		'[aria-label="Organization"] [data-tab-item="org-header-overview-tab"] a', // "Overview" tab on organization profile
+		'[aria-label="Organization"] a[data-tab-item="i0overview-tab"]', // "Overview" tab on organization profile (Global navigation update)
 		'a[data-hovercard-type="organization"]', // Organization name on repo header + organization list on user profile
```

Comment:
I think the solution is:

```suggestion
		// User repos
		`a:is([href^="/"], [href^="${location.origin}"])[href$="?tab=repositories"]`,

		// Organization repos
		`a:is([href^="/orgs/"], [href^="${location.origin}/orgs/"])[href$="/repositories"]`,
```

Reply:
Did you mean we drop support for these?
<img width="296" alt="Screenshot 2023-04-16 at 20 16 39" src="https://user-images.githubusercontent.com/44045911/232309379-a4e92e18-0ca2-4a9d-852f-d30663738a53.png">
<img width="326" alt="Screenshot 2023-04-16 at 20 16 57" src="https://user-images.githubusercontent.com/44045911/232309381-1bad2085-85fc-4ce5-8601-9f1d2236d74a.png">